tests/form.js

```diff
@@ -45,3 +46,28 @@ test('Form Generation, Again Again', t => {
     t.is(typeof rendered.html, 'string', 'HTML is a string');
   });
 });
+
+test('Form Generation, Again Again Again', t => {
```

Comment:
Sorry...was copying the naming from _whomever_ might have named the ones before it.